packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -218,15 +218,29 @@ class _TabView extends StatefulWidget {
 
 class _TabViewState extends State<_TabView> {
   List<Widget> tabs;
+  List<FocusScopeNode> tabFocusNodes;
 
   @override
   void initState() {
     super.initState();
     tabs = new List<Widget>(widget.tabNumber);
+    tabFocusNodes = new List<FocusScopeNode>.generate(
+      widget.tabNumber,
+      (_) => new FocusScopeNode(),
```

Comment:
we should give the type and name of the argument here, so that people browsing the code don't have to look it up